entrypoint.sh

```diff
@@ -12,9 +12,17 @@ cf api "$INPUT_CF_API"
 cf auth "$INPUT_CF_USERNAME" "$INPUT_CF_PASSWORD"
 cf target -o "$INPUT_CF_ORG" -s "$INPUT_CF_SPACE"
 
-if [[ -z "$INPUT_CF_COMMAND" ]]; then
-    cf v3-zdt-push -f "$MANIFEST"
```

Comment:
This is no longer the intended way to push an app?  I took a look [in the docs](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html#deploy), are we using CF CLI v7 now?  Is this script just for us to use or customers too?

Reply:
The `cf v3-zdt-push` command doesn't let you specify a manifest file using the `-f` flag, so I removed that and just replaces it with `cf push` which does. You can still use `cf v3-zdt-push` for a zero downtime deploy by [specifying a specific command](https://github.com/cloud-gov/cg-cli-tools/blob/add-vars-support/action.yml#L25-L27) when setting up the workflow that uses this action.

Reply:
Got it, thank you for explaining and that's cool that you can still specify/override if need be!